packages/waku/src/lib/rsc/worker-api.ts

```diff
@@ -30,6 +31,7 @@ export type MessageReq =
 export type MessageRes =
   | { type: 'full-reload' }
   | { type: 'hot-import'; source: string }
+  | { type: 'module'; result: TransformResult }
```

Comment:
It's ambiguous, can we call it `module-import` instead of `module`?

Reply:
Why not, let me try.